code/modules/hydroponics/gene_modder.dm

```diff
@@ -17,10 +17,10 @@
 	var/operation = ""
 	var/max_potency = 50 // See RefreshParts() for how these work
 	var/max_yield = 2
-	var/min_production = 12
+	var/min_production = 9
 	var/max_endurance = 10 // IMPT: ALSO AFFECTS LIFESPAN
-	var/min_wchance = 67
-	var/min_wrate = 10
+	var/min_wrate = 8
+	var/min_wchance = 24
```

Comment:
The weed chance being dropped from 67 to 24 is quite drastic

Reply:
Isn't that the percentage chance for a plant to get overran with weeds every cycle once there are too many weeds in the tray? A one in for chance of that happening is still pretty high especially since production rate is capped at 9 for the machine.

Reply:
An over 50% weed chance is way too much, players shouldn't be punished for having bad luck with mutations and then needing to stick with it, if they don't recieve upgrades (mutations like to give plants 67% weed chance, which is frustrating if you can't change it in further generations of the plant)

Reply:
Mmm, seems fair